docs/operations/metrics.md

```diff
@@ -72,11 +69,11 @@ Available Metrics
 |`segment/scan/pending`|Number of segments in queue waiting to be scanned.||Close to 0|
 |`query/segmentAndCache/time`|Milliseconds taken to query individual segment or hit the cache (if it is enabled on the Historical process).|id, segment.|several hundred milliseconds|
 |`query/cpu/time`|Microseconds of CPU time taken to complete a query|Common: dataSource, type, interval, hasFilters, duration, context, remoteAddress, id. Aggregation Queries: numMetrics, numComplexMetrics. GroupBy: numDimensions. TopN: threshold, dimension.|Varies|
-|`query/count`|number of total queries|This metric is only available if the QueryCountStatsMonitor module is included.||
-|`query/success/count`|number of queries successfully processed|This metric is only available if the QueryCountStatsMonitor module is included.||
-|`query/failed/count`|number of failed queries|This metric is only available if the QueryCountStatsMonitor module is included.||
-|`query/interrupted/count`|number of queries interrupted due to cancellation.|This metric is only available if the QueryCountStatsMonitor module is included.||
-|`query/timeout/count`|number of timed out queries.|This metric is only available if the QueryCountStatsMonitor module is included.||
+|`query/count`|Number of total queries|This metric is only available if the QueryCountStatsMonitor module is included.||
```

Comment:
Number of total reads awkwardly is this the "total number of queries"?

Reply:
```suggestion
|`query/count`|Total number of queries|This metric is only available if the QueryCountStatsMonitor module is included.||
```